sys-devel/automake/automake-1.15.1-r1.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```

Comment:
you'll likely have to pull this down to EAPI 5 due to weird toolchain/basesystem policies